src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -1090,7 +1050,7 @@ def run_dm_activities_with_ior(self, tool, create_dataset=False, pool=None, cont
             # Create pool2
             pool2 = self.get_pool()
             # Use dfuse as a shared intermediate for serialize + deserialize
-            dfuse_cont = self.create_cont(pool, oclass=self.ior_cmd.dfs_oclass.value)
+            dfuse_cont = self.add_container(pool, oclass=self.ior_cmd.dfs_oclass.value)
```

Comment:
(pylint-assignment-from-no-return) Assigning result of a function call, where the function has no return
(pylint-unexpected-keyword-arg) Unexpected keyword argument 'oclass' in method call